app/controllers/articles_controller.rb

```diff
@@ -34,17 +42,9 @@ def update
     end
   end
 
-  def destroy
-    @article = Article.find(params[:id])
-    @article.destroy
-
-    redirect_to root_path
-  end
-  
-
   private
 
   def article_params
-    params.require(:article).permit(:title, :body)
+    params.require(:article).permit(:title, :body, :status)
```

Comment:
@JuanMaGiraldo what the **require** and **permit** do? and why are they important?

Reply:
The "require" method ensures that a specific parameter is present, otherwise it will throw an exception. The "permit" method is a list of allowed attributes, these can be optional. These methods help with argument handling. By not using these, it would be necessary to make a query for each attribute "params [: obj] [: attributte]", the more attributes the object has, this can get much more complicated. Also, the user can be malicious and send an "admin = true", that's where the Strong Parameters come in and the importance of these methods.